.skillsaw/plugindocs_rule.py

```
@@ -48,7 +48,7 @@ def check(self, context: RepositoryContext) -> List[RuleViolation]:
         original_content = index_path.read_text()
         try:
             result = subprocess.run(
-                ["skillsaw", "docs", "-o", "docs/"],
+                ["skillsaw", "docs", "-o", "docs/", "--theme", "crimson-red"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)